doc/api/worker_threads.md

```diff
@@ -375,7 +376,7 @@ added: v10.5.0
 * `transferList` {Object[]}
 
 Send a message to the worker that will be received via
-[`require('worker_threads').on('workerMessage')`][].
+[`require('worker_threads').parentPort.on('message')`][].
```

Comment:
I think in this case the norm is to link `worker_threads.parentPort` and `'message'` separately, to something like

```markdown
received via [`worker_threads.parentPort`][]'s [`'message'`][`port.on('message')`] event.
```